checkov/terraform/checks/resource/aws/EKSControlPlaneLogging.py

```diff
@@ -19,7 +19,8 @@ def scan_resource_conf(self, conf):
         :return: <CheckResult>
         """
         log_types = ["api", "audit", "authenticator", "controllerManager", "scheduler"]
-        if "enabled_cluster_log_types" in conf.keys() and conf["enabled_cluster_log_types"][0] is not None \
+        if "enabled_cluster_log_types" in conf.keys() and conf["enabled_cluster_log_types"] and \
+                conf["enabled_cluster_log_types"][0] is not None \
                 and all(elem in conf["enabled_cluster_log_types"][0] for elem in log_types):
```

Comment:
```suggestion
        log_types = conf.get("enabled_cluster_log_types")
        if log_types and isinstance(log_types, list) and isinstance(log_types[0], list) \
                and all(elem in log_types[0] for elem in log_types):
```
how about this, then we also check, if it is really a list, before trying to iterate over it 😄